examples/starter-node-example/index.js

```diff
@@ -0,0 +1,27 @@
+const http = require('http');
+const port = 3000;
+// Define the hostname as '0.0.0.0', which means the server will bind to all available network interfaces
+const hostname = '0.0.0.0';
+
+// Create the HTTP server
+const server = http.createServer((req, res) => {
+  // Check if the request method is 'GET' and the requested URL is '/'
+  if (req.method === 'GET' && req.url === '/') {
+    res.statusCode = 200;
+    res.setHeader('Content-Type', 'text/plain');
+    // Send the response body with a congratulatory message
+    res.end('Congratulations on interacting with your BrightSign running a simple node http application!');
+  
+  } else {
+    res.statusCode = 404;
+    res.setHeader('Content-Type', 'text/plain');
+    res.end('404 Not Found');
```

Comment:
Update to include an actionable message in response. 

404 is the defined `statusCode`. 

Though here could indicate "Route not found. Try a GET Request to /`